src/h3/api/_api_template.py

```diff
@@ -1,5 +1,5 @@
-import h3.h3core as h3core
-import h3.h3utils as u
+import h3.memview_int as mvi
```

Comment:
Nit: I'm not thrilled with this naming chance - it feels to me like it loses some useful abstraction, unless the `memview` name is conventional for direct access to memory.

Reply:
Memory view is the name of the actual python object we're returning: https://docs.python.org/3/c-api/memoryview.html
It is indeed just a lightweight Python wrapper around memory.

That follows the convention of `set` or `array` as the collection objects used in other APIs.

Memory views don't require any outside dependencies, but most users would expect something like a numpy array, which is why we provide that interface.